x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/views/config_viewer/metric.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export class Metric {
+  constructor(name, className, value) {
+    this.name = name;
+    this.className = className;
+    this.value = value;
+  }
+}
```

Comment:
Seeing how this class is being used, I'm wondering if it makes sense / is more idiomatic to just make this class a React component instead? It would essentially render this bit:

https://github.com/elastic/kibana/pull/18597/files#diff-3e6d8fcbe80019267de66cb877aa376bR62

Reply:
Originally these objects were just created inline, then I added a class to formalize their props. In hindsight I think the whole thing was just a smell asking for a component, I will do this.